### PR TITLE
Initial setup with Laravel and Homestead

### PR DESCRIPTION
**Pull request by martindilling**
Not much happening, so now I'm just starting with something ;)
The initial setup of Laravel 5.1 and Homestead.

Comment here and let me know if you want it merged or if you have any complaints, feedback, requests, etc. to this PR :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/vendor
+/node_modules
+Homestead.yaml
+Homestead.json
+.env
```
Comment (Munk91) [inline .gitignore]: Man kunne godt overveje om bower_components og composer.phar skal med her også.

Comment (martindilling) [inline .gitignore]: Good point, will add them tonight :)

Comment (JessNielsen) [inline .gitignore]: og evt .idea da de fleste her vel er phpstorm addicted

2015-12-01 16:15 GMT+01:00 Martin Dilling-Hansen <notifications@github.com>:

> In .gitignore
> <https://github.com/collabexperiment/platform/pull/6#discussion_r46289680>
> :
>
> > @@ -0,0 +1,5 @@
> > +/vendor
> > +/node_modules
> > +Homestead.yaml
> > +Homestead.json
> > +.env
>
> Good point, will add them tonight :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/collabexperiment/platform/pull/6/files#r46289680>.
>



-- 
Jess Nielsen

Otto Banners Vej 16, 9300 Sæby
M 21 30 13 41

Comment (martindilling) [inline .gitignore]: That belongs in your own gitignore :)

**Pull request merged**